news/2023/2023-09-25-osulazer-updates-september-24.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "osu!(lazer) Updates: September 24"
+date: 2023-09-25 21:00:00 +0000
+---
+
+Marvel at the never-ending stream of usability improvements.
+
+![](/wiki/shared/news/2023-09-25-osulazer-updates-september-24/banner.jpg)
+
+Things are progressing faster towards the goal of leaderboards by the end of the year, but so is the to-do list of getting there. Find out all about it in this video here or read the executive summary below:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/SWEE8XlGluI" frameborder="0" allowfullscreen></iframe>
+</div>
+
+### Changes
+
+- **Velocity change visualisation toggle in the editor.** In osu!mania and osu!taiko, the speed of objects can be toggled to be constant or reflect gameplay.
+- **-10 approach rate for the Difficulty Adjust mod.** Courtesy of reading expert [-GN](https://osu.ppy.sh/users/895581), you may now attempt and fail something he does with ease.
+- **Removed slider head movement.** No more misses when hitting a slider late. This was a consequence of the new slider "snake out" feature, but has now been fixed.
+- **Beatmap attribute display in the mod select overlay.** Star rating, BPM, and other info can be seen on the left-side "details" tab, but when trying to select mods, a huge (albeit fabulous) overlay would rudely block the view. That is now fixed along with other quality-of-life improvements.
+- **Missing beatmap download on replay import.** Not entirely automatic, but the game will present you with the correct beatmap (in a new *nano-sized* beatmap card) if you don't have the map for a replay your friend sent you.
```

Comment:
(it's automatic if you have the "automatically download missing beatmap" toggle enabled)